modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -206,7 +223,7 @@ START_TEST(ILCreateFromPath)
 START_TEST(PIDL)
 {
     CCoInit ComInit;
-    LPITEMIDLIST pidl;
+    LPITEMIDLIST pidl, pidl2;
```

Comment:
Suggestion: move the pidl2 variable declaration inside the block where it's used.